lib/iris/coord_systems.py

```diff
@@ -935,19 +971,20 @@ def __init__(
         Kwargs:
 
             * central_lat
-                    The latitude of "unitary scale".
+                    The latitude of "unitary scale".  Defaults to 39.0
 
             * central_lon
-                    The central longitude.
+                    The central longitude.  Defaults to -96.0
 
             * false_easting
-                    X offset from planar origin in metres.
+                    X offset from planar origin in metres.  Defaults to 0
```

Comment:
@pp-mo and I have agreed offline that this file needs a quick check through to make sure docstrings consistently make it clear that the defaults are `float` types, since different lines containing integers and floats may be misinterpreted as actual different behaviour.